prismic-model/src/webcomics.ts

```diff
@@ -36,14 +36,6 @@ const webcomics: CustomType = {
         series: link('Series', 'document', ['webcomic-series']),
       }),
     },
-    Deprecated: {
-      publishDate: {
```

Comment:
Won't this remove it from documents in Prismic?

Or is it never set on webcomics?

Reply:
Good point, it does keep the data, until the document is published again in which case the data would be blatted. 

Maybe we just make sure that editors know that this will override the rendering but not the ordering. [See here](https://prismic.io/docs/technologies/order-your-results-javascript#order-by-multiple-fields) for why we can't order by a merged value of these fields - it's basically a tie breaker style ordering.

Reply:
To be honest this type is now disabled anyway - so the editors can't use it.